src/components/breadcrumb/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import classnames from 'classnames'
+
+export interface BaseBreadcrumbProps {
+  children: React.ReactNode
+  className?: string
+  listItemProps?: JSX.IntrinsicElements['li']
+}
+interface BreadcrumbProps extends BaseBreadcrumbProps {
+  current?: boolean
+}
+
+export const Breadcrumb = (props: BreadcrumbProps): JSX.Element => {
```

Comment:
Good catch @suzubara on adding the JSX intrinsic props. Adding this comment here to remind myself.